app/api/edlinker/tasks.rb

```diff
@@ -11,14 +11,12 @@ def declared_params
   namespace :users do
     get ':user_id/tasks' do
       current_user = User.find(params[:user_id])
-      current_user.tasks.includes(:messages).map do |task|
-        assignment = task.messages.first
+      current_user.tasks.map do |task|
         {
-            author: assignment.author,
             description: task.description,
-            posted_at: task.created_at,
-            body: assignment.body,
-            link: assignment.url,
+            status: task.status,
+            url: task.url,
```

Comment:
name of this field should be urls, at least. Also you need to be sure that response format is always the same, and frontend doesn't have to work with a few variants of responses